packages/vue/src/tracing.ts

```diff
@@ -46,6 +46,25 @@ function finishRootSpan(vm: VueSentry, timestamp: number, timeout: number): void
   }, timeout);
 }
 
+// Find if the current component exists in the provided `TracingOptions.trackComponents` array option.
+/**
+ *
+ */
+export function findTrackComponent(trackComponents: string[], formattedName: string): boolean {
+  /**
+   *
+   */
+  function extractComponentName(name: string): string {
+    return name.replace(/^<([^\s]*)>(?: at [^\s]*)?$/, '$1');
```

Comment:
Based on what you suggested @Lms24, just an added non-capture group. The ReDoS Checker passes.

Reply:
Sounds good to me, thanks!